include/uapi/sound/sof/tokens.h

```diff
@@ -97,7 +98,6 @@
 #define SOF_TKN_COMP_SINK_PIN_BINDING_WNAME	413
 #define SOF_TKN_COMP_SRC_PIN_BINDING_WNAME	414
 
-
```

Comment:
spurious change

Reply:
Looks like 9331aa0feb774 fixed this for me.